### PR TITLE
Update Desktop Readme

### PR DESCRIPTION
**Pull request by stevenGravy**
Expands links to include all Windows Server types
Modify powershell that works with all versions.

### CODE REVIEW

File: lib/srv/desktop/README.md [hunk 1]
```diff
@@ -1,22 +1,34 @@
-# Windows Desktop Access Beta
+# Windows Desktop Access VM Setup
 
-## How to set up Desktop Access on Windows
+## How to set up Windows Server for Desktop Access
 
-### Install Windows Server 2012
+These instructions install a Domain Controller and Certificate Serviers on a Windows Server that
```
Comment (zmb3) [inline lib/srv/desktop/README.md:5]:
```suggestion
These instructions install a Domain Controller and Certificate Servers on a Windows Server that
```

File: lib/srv/desktop/README.md [hunk 2]
```diff
@@ -1,22 +1,34 @@
...
 
-### Set up Active Directory
+### Set up Domain Controller with Active Directory Domain Services
 
 #### AD DS
 
 First, we need to install Active Directory Domain Services (AD DS). Save the following file as `domain-controller.ps1`,
-replacing `$domain` with your desired domain name.
+replacing `$domain` with your desired domain name.  The `Default` setting for `ForestMode` and `DomainMode` will match to the server version.  
+See Msft instructions for all options.
```
Comment (zmb3) [inline lib/srv/desktop/README.md:31]:
```suggestion
```

Either provide a link, or don't mention this. Saying "see some other instructions" without linking to any other instructions is equivalent to saying "just google it if you get stuck"

**Review by zmb3 — COMMENTED** [review]
I'm more inclined to just remove this file as it has a lot of duplication with our docs and glosses over some important details for real-world set ups.

File: lib/srv/desktop/README.md (hunk 2 above)
Comment (stevenGravy) [reply lib/srv/desktop/README.md]: @zmb3 added link to powershell instructions that gives all parameter options

**Pull request closed without merging**